src/modules/didIpfsHelper.js

```diff
@@ -0,0 +1,163 @@
+const ChluDID = require('chlu-did/src')
```

Comment:
This is getting duplicated from the chlu-ipfs, right? Just clarifying.

Reply:
It's not duplicated, I made changes here. But most of the code is a duplicate, so when we refactor the DID stuff we will fix this and make sure we don't have repeated code.